### DISCUSSION TITLE
Addictions

### DISCUSSION THREAD

**Discussion by AurelieCatena** [littlesera/LSCG#381] [Ideas]
**Addictions:**
- Orgasm
- Pain
- Fluids
- Alcohol
- Drugs
- Aphrodisiacs
- ...

**Effects:**
You must get a fix of what you are addicted to every _x_ hours. If you don't, you suffer from withdrawal symptoms. 
The more severe your addiction, the more often you need your fix.

**Withdrawal**
The longer you wait after you enter withdrawal, the stronger the symptoms grow:

1. Just annoying private messages
2. Public messages that indicate your shivering, or sweating, or whatever
3. Your vision get blurred
4. You stumble and fall on your knees at random time. Standing back up requires a mini-game.
5. You are slowed down (when leaving a room)
6. You can no longer tie others
7. You are sweating so much that you must be naked
8. You freeze (like hypnosis)

**Healing**
Each 10 minutes you spend in the last stage of withdrawal reduces your addiction by one point.

**Comment by anchillabc** [comment]
I must say... having known people who struggled with addiction in their daily life, I do not think this is a good idea to add to the game as it may trigger very real and very problematic issues they are already dealing with.

**Reply by AurelieCatena** [reply]
That could also be said of other activities like sexual assault, rape, kidnapping, strangulation, etc. which is what this mod is about...

**Comment by littlesera** [comment]
A slightly different angle for these mechanics would be to establish needs while under hypnosis/brainwashing.
As a note, all LSCG features are opt-in by default. Surprising anyone with sensitive subject matter like all the above is hugely problematic and I try to avoid it as much as possible.
